csharp/ql/src/semmle/code/csharp/security/dataflow/flowsinks/Remote.qll

```diff
@@ -8,7 +8,7 @@ private import ExternalLocationSink
 private import Html
 private import semmle.code.csharp.security.dataflow.XSS
 private import semmle.code.csharp.frameworks.system.web.UI
-import semmle.code.csharp.frameworks.ServiceStack::Sinks
+private import semmle.code.csharp.frameworks.ServiceStack::Sinks
```

Comment:
This can be removed if the `ServiceStackRemoteRequestParameter` extends `ExternalLocationSink`, can't it?

Reply:
Fixed.